client/pom.xml

```diff
@@ -45,6 +45,26 @@
             <groupId>com.sun.jersey</groupId>
             <artifactId>jersey-client</artifactId>
         </dependency>
+
+        <!-- CXF still using jakarta.ws.rs-api 2.x, we have here 3.X  e.g. package javax.ws.rs.  -->
+        <dependency>
+            <groupId>org.apache.cxf</groupId>
+            <artifactId>cxf-rt-rs-client</artifactId>
+        </dependency>
+        <!--  Import JAX-RS 2.0/javax.ws.rs. from different artifactId  -->
+        <dependency>
+            <groupId>javax.ws.rs</groupId>
+            <artifactId>javax.ws.rs-api</artifactId>
+            <version>2.1.1</version>
+        </dependency>
+        <!-- This is supposed to come from spring boot, not needed for now
```

Comment:
If we don't need to explicitly specify this dependency, `jackson-jaxrs-json-provider`, here than we can drop it in my opinion